posthog/models/filters/mixins/common.py

```diff
@@ -408,3 +408,13 @@ def target_entity_type(self) -> Optional[str]:
     @include_dict
     def entity_type_to_dict(self):
         return {"entity_type": self.target_entity_type} if self.target_entity_type else {}
+
+
+class SelectedIntervalMixin(BaseParamMixin):
```

Comment:
Should we perhaps clearly indicate with naming or comments that this is stickiness-specific?